source/_integrations/tuya.markdown

```diff
@@ -2,19 +2,18 @@
 title: Tuya
 description: Instructions on how to setup the Tuya hub within Home Assistant.
 ha_category:
-  - Hub
   - Climate
   - Cover
   - Fan
   - Light
   - Scene
   - Switch
 ha_iot_class: Cloud Polling
-ha_release: 0.74
+ha_release: 2021.9.7
 ha_config_flow: true
-ha_domain: tuya
+ha_domain: Tuya
```

Comment:
All domains are lowercase. It's the directory name in HA.

```suggestion
ha_domain: tuya
```

Reply:
fixed.